internal/kubernetes/operator/project/project.go

```diff
@@ -446,28 +446,7 @@ func buildIntegrations(intProvider store.IntegrationLister, projectID, targetNam
 }
 
 func getIntegrationType(val atlasv2.ThridPartyIntegration) string {
```

Comment:
Key simplicity of the new SDK

Reply:
I think we should delete this method it doesn't abstract anything anymore